roles/postgresql/tasks/packages_client.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+---
+- name: Disable dnf module postgresql
+  command: dnf module disable postgresql -y
+  args:
+    warn: no
+  when: ansible_distribution_major_version | int > 7
+  register: dnf_result
+  changed_when: '"Disabling modules" in dnf_result.stdout'
+  notify: yum makecache
+
+- import_tasks: public_repo.yml
+
+- name: Install PostgreSQL adapter for python
+  yum:
+    name: "{{ postgresql_default_packages[packages_version] }}"
```

Comment:
Here you use the variable `postgresql_default_packages` which contains packages for PostgreSQL server not for clients.
You should add a variable `postgresql_client_default_packages` with packages for clients. Later we will add the `postgresql-jdbc` package in this new variable.

Reply:
Done